artikel/2015-07-31-Hitzewelle-Sommerspaß-oder-Klimawandel.md

```diff
@@ -0,0 +1,34 @@
+#Hitzewelle: Sommerspaß oder Klimawandel?
+
+http://www.dw.com/de/hitzewelle-sommerspa%C3%9F-oder-klimawandel/a-18616694
+
+der Artikel "Hitzewelle: Sommerspaß oder Klimawandel?" spricht ein
+interessantes und wichtiges Thema an, weil wir diesen Sommer so viel 
+Hitzwelle in Deutschland gehabt haben.
+
+Der Artikel geht um Klimawandel und die Möglichkeit, ob es existiert. In 
+2015 gab es rekorde Temperaturen und häufiger Hitzwelle in Deutschland. 
+Im Juli 2015 gab es einen neuen Hitzrekord in Bayern mit 40,3 Grad Celsius 
+gemessen. Obwohl das heiße Wetter schön für Deutschen ist, finden die 
+meisten, dass ihre Wohnungen nicht für das heiße Wetter gut gebaut sind.
+
+Einerseits freuen die meisten Deutschen auf das heiße Wetter, andereseits 
+haben manche Deutschen Angst vor dem Klimawandel, denn sie glauben, dass 
+Menschen den Planeten erwärmen. Eine Expertin, die Dim Coumou heißt, 
```

Comment:
This last part is strange. `Menschen den Planeten erwärmen` is like "People heating up the planet". You miss the relation between people and climate change somehow, like "because of the people". Even more, your reasoning starts with `denn sie glauben` (because they believe), which would require you to go for subjunctive construction in your conditional. It would get slightly complicated then:

    denn sie glauben, es sei der Mensch, der dem Planeten schade

Note that `schade` here is the subjunctive form of `schaden` (to hurt). `Der Mensch` in its singular here refers to sth like "Mankind".